python/ray/experimental/state/common.py

```diff
@@ -416,6 +418,12 @@ class NodeState(StateSchema):
     node_name: str = state_column(filterable=True)
     #: The total resources of the node.
     resources_total: dict = state_column(filterable=False)
+    #: The time when the node (raylet) starts.
```

Comment:
When we print out to users on CLI, do you think if we should convert all the timestamps to human readable values?

Reply:
we should. I think it is same for task events. 

I think we need a way to distinguish API return value / display values. Maybe you can consider adding a task to fix this?

Reply:
Right, I guess we could do the formatting upon printing? If I use `list_tasks` programmatically, I might still have the value as `ms`, but I think we should make it readable for display. 
Tracked here: https://github.com/ray-project/ray/issues/31876

Reply:
yeah. I think we can either process that from the CLI, or we can have something like view_callback for each field (which is invoked when it is printed)